cmd/nerdctl/helpers/testing.go

```diff
@@ -30,3 +41,97 @@ func CreateBuildContext(t *testing.T, dockerfile string) string {
 	assert.NilError(t, err)
 	return tmpDir
 }
+
```

Comment:
As explained in the commit message, we do currently have a mix of platform specific test files (eg: _platform_test - along with dependencies that may be platform limited too) and tests that call `t.Skip` on certain platforms.

These helpers here are moved around into the main helper file so that we can reduce a little bit the platform specific test files.

Either way, all of that has nothing to do here - unless they are used in live code, they should be ultimately moved to `pkg/testutil` (or subsumed / removed). Work for another PR.